build.sbt

```diff
@@ -1,17 +1,15 @@
 
-lazy val scala12Version = "2.12.8"
-
 inThisBuild(
   (homepage := Some(url("https://github.com/gemini-hlsw/sbt-gsp"))) +: gspPublishSettings
 )
 
-addSbtPlugin("de.heikoseeberger" % "sbt-header" % "5.2.0")
+addSbtPlugin("de.heikoseeberger"         % "sbt-header"   % "5.2.0")
+addSbtPlugin("io.github.davidgregory084" % "sbt-tpolecat" % "0.1.7")
 
 lazy val sbtGsp = (project in file("."))
+  .enablePlugins(SbtPlugin)
+  .enablePlugins(AutomateHeaderPlugin)
   .settings(
-    name         := "sbt-gsp",
-    scalaVersion := scala12Version,
```

Comment:
Since it's an sbt plugin it should use whatever version of Scala sbt is using, so we don't need to specify it.